README.md

```diff
@@ -103,13 +92,12 @@ The extension does most of the heavy lifting when it comes to gathering and prep
 
 Depending on the extension version you are using, you could have a different PHP API client version powering the extension's backend functionality.
 
-| Extension Version | API Client Version                                                        |
-|-------------------|---------------------------------------------------------------------------|
-| v1.x              | [1.28.0](https://github.com/algolia/algoliasearch-client-php/tree/1.28.0) |
-| v2.x              | [2.5.1](https://github.com/algolia/algoliasearch-client-php/tree/2.5.1)   |
+| Extension Version | API Client Version     |
+|-------------------|------------------------|
 | v3.x              | [2.5.1](https://github.com/algolia/algoliasearch-client-php/tree/2.5.1)   |
 | v3.6.x            | [3.2.0](https://github.com/algolia/algoliasearch-client-php/tree/3.2.0)   |
-| >= v3.11.0        | [3.3.2](https://github.com/algolia/algoliasearch-client-php/tree/3.3.2)   |
+| v3.11.0          | [3.3.2](https://github.com/algolia/algoliasearch-client-php/tree/3.3.2)   |          
+| >=v3.14.x         | `4.0.x`                |
```

Comment:
@mrahman3177 I'm thinking we should provide a link to the 4.0.x Github. We can reference the [latest beta](https://github.com/algolia/algoliasearch-client-php/tree/4.0.0-beta.12) or the [`next` branch](https://github.com/algolia/algoliasearch-client-php/tree/next).